### ISSUE TITLE
Reimplement about tab

### ISSUE THREAD

**Issue by aubymori** [Rehike/Rehike#409]
As of now (2023/11/03), YouTube is testing an experiment that removes the About tab and instead puts that information in a dialog which can be accessed by clicking the description preview on a channel page.

![image](https://github.com/Rehike/Rehike/assets/44238627/4d23f127-dc92-4c6f-89b3-7d27ed5be6c2)

This is bad for Rehike because it means that the About tab is just entirely gone.

![image](https://github.com/Rehike/Rehike/assets/44238627/63c41fe4-41ef-40bd-810e-76fb728d9087)

Even attempting to go to the about page manually causes a fatal error.

```
YukisCoffee\CoffeeRequest\Exception\PromiseAllException: Promise 1 rejected in Promise::all() call with reason ""
  at YukisCoffee\CoffeeRequest\Util\PromiseAllBase::handlePromiseRejection(YukisCoffee\CoffeeRequest\Promise)(rehike://modules/YukisCoffee/CoffeeRequest/Util/PromiseAllBase.php:90)
  at YukisCoffee\CoffeeRequest\Util\PromiseAllBase::YukisCoffee\CoffeeRequest\Util\{closure}(Rehike\Exception\Network\InnertubeFailedRequestException)(rehike://modules/YukisCoffee/CoffeeRequest/Util/PromiseAllBase.php:130)
  at YukisCoffee\CoffeeRequest\Promise::reject(Rehike\Exception\Network\InnertubeFailedRequestException)(rehike://modules/YukisCoffee/CoffeeRequest/Promise.php:367)
  at YukisCoffee\CoffeeRequest\Promise::YukisCoffee\CoffeeRequest\{closure}(Rehike\Exception\Network\InnertubeFailedRequestException)(rehike://modules/YukisCoffee/CoffeeRequest/Promise.php:387)
  at Rehike\Network::Rehike\{closure}(YukisCoffee\CoffeeRequest\Network\Response)(rehike://modules/Rehike/Network.php:136)
  at YukisCoffee\CoffeeRequest\Promise::resolve(YukisCoffee\CoffeeRequest\Network\Response)(rehike://modules/YukisCoffee/CoffeeRequest/Promise.php:285)
  at YukisCoffee\CoffeeRequest\Util\QueuedPromiseResolver::finish()(rehike://modules/YukisCoffee/CoffeeRequest/Util/QueuedPromiseResolver.php:57)
  at YukisCoffee\CoffeeRequest\Loop::finishQueuedPromises()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:240)
  at YukisCoffee\CoffeeRequest\Loop::cleanup()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:230)
  at YukisCoffee\CoffeeRequest\Loop::run()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:84)
  at YukisCoffee\CoffeeRequest\Loop::finishQueuedPromises()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:244)
  at YukisCoffee\CoffeeRequest\Loop::cleanup()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:230)
  at YukisCoffee\CoffeeRequest\Loop::run()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:84)
  at YukisCoffee\CoffeeRequest\Loop::finishQueuedPromises()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:244)
  at YukisCoffee\CoffeeRequest\Loop::cleanup()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:230)
  at YukisCoffee\CoffeeRequest\Loop::run()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:84)
  at YukisCoffee\CoffeeRequest\CoffeeRequest::run()(rehike://modules/YukisCoffee/CoffeeRequest/CoffeeRequest.php:121)
  at Rehike\Network::run()(rehike://modules/Rehike/Network.php:230)
  at Rehike\Controller\core\HitchhikerController::get(Rehike\YtApp, "", Rehike\ControllerV2\RequestMetadata)(rehike://controllers/core/HitchhikerController.php:125)
  at Rehike\ControllerV2\GetControllerInstance::_cv2WrappedCallControllerMethod("get", array<0>)(rehike://modules/Rehike/ControllerV2/GetControllerInstance.php:51)
  at Rehike\ControllerV2\GetControllerInstance::__call("get", array<0>)(rehike://modules/Rehike/ControllerV2/GetControllerInstance.php:74)
  at Rehike\ControllerV2\Router::pointerHandler("channel", "get")(rehike://modules/Rehike/ControllerV2/Router.php:213)
  at Rehike\ControllerV2\Router::baseRequestMethod(array<24>, "get")(rehike://modules/Rehike/ControllerV2/Router.php:164)
  at Rehike\ControllerV2\Router::get(array<24>)(rehike://modules/Rehike/ControllerV2/Router.php:26)
  at require("C:\Users\Aubrey\src\Rehike\router.php")(rehike://router.php:77)
  at Rehike\Boot\Bootloader::postboot()(rehike://modules/Rehike/Boot/Bootloader.php:56)
  at Rehike\Boot\Bootloader::startSession()(rehike://modules/Rehike/Boot/Bootloader.php:23)
  at {main}(rehike://index.php:72)
```

It is worth noting that while opening the about dialog from another channel tab will perform a browse request with a continuation, requesting `/<channel>/about` with Polymer will automatically open the dialog which is embedded in the response. This means that hopefully no extra requests should be needed.

**Comment by ephemeralViolette** [comment]
upload yti response with experiment

**Comment by aubymori** [comment]
> upload yti response with experiment

[Here](https://github.com/Rehike/Rehike/files/13254071/kosmicd12.json). From the YouTube channel [Kosmic](https://www.youtube.com/@Kosmicd12/about). Should have most about information that is available.

**Comment by ephemeralViolette** [comment]
eee60d9?

**Comment by aubymori** [comment]
> [eee60d9](https://github.com/Rehike/Rehike/commit/eee60d9bfb92b8b41b9b23ddf18d72e8824bce1f)?

![image](https://github.com/Rehike/Rehike/assets/44238627/5d9c5116-42fd-4944-a5e7-88c6ade17da9)
![image](https://github.com/Rehike/Rehike/assets/44238627/d488f42e-b5dd-4108-864c-fdf22e4f682c)
same error as before
![image](https://github.com/Rehike/Rehike/assets/44238627/3d62afce-304d-4ed2-9bb4-46232c742358)
version correct too

**Comment by ephemeralViolette** [comment]
6f72f84?

**Comment by aubymori** [comment]
still exact same behavior

**Comment by ephemeralViolette** [comment]
hopefully fixed in 8c45fe9
